trufflehog.yml

```
@@ -0,0 +1,16 @@
+    exclude_paths:
+      - "*.lock"
+      - "**/package-lock.json"
+      - "**/yarn.lock"
+      - "**/node_modules/**"
+      - "**/__pycache__/**"
+      - "**/.venv/**"
+      - "**/target/**"
+      - "**/.git/**"
+    detectors:
+      - allowlist: false
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)